### DISCUSSION TITLE
NJU 软件分析 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#497] [Announcements]
# NJU 软件分析 - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%BC%96%E8%AF%91%E5%8E%9F%E7%90%86/NJU-SoftwareAnalysis/

**Comment by sarendis56** [comment]
呜呜呜，更新的比我学得快

**Comment by Qi-Zhan** [comment]
对于学完了本课程还想要进一步了解的同学，我极力推荐 [spa](https://cs.au.dk/~amoeller/spa/) 这本书，我自己也正在学习。这本书讲的比南大课程更为全面但不像其那么好理解，我个人认为更偏理论适合进阶

**Comment by zzzfreeuser** [comment]
求相关交流群
